src/components/App.tsx

```diff
@@ -1562,10 +1572,17 @@ class App extends React.Component<AppProps, AppState> {
       selectGroupsForSelectedElements(
         {
           ...this.state,
-          isLibraryOpen:
-            this.state.isLibraryOpen && this.device.canDeviceFitSidebar
-              ? this.state.isLibraryMenuDocked
-              : false,
+          // keep sidebar (presumably the library) open if it's docked and
+          // can fit.
+          //
+          // Note, we should close the sidebar only if we're dropping items
+          // from library, not when pasting from clipboard. Alas.
+          openSidebar:
+            this.state.openSidebar &&
+            this.device.canDeviceFitSidebar &&
+            this.state.isSidebarDocked
+              ? this.state.openSidebar
+              : null,
```

Comment:
Here you are checking whether sidebar is open and if view port could fit and docked then only keep open, how the clipboard pasting vs dropping is distinguished as mentioned in comment ?

Reply:
it's not. This is no change from production. We can fix it in a future PR. It's an edge case that doesn't really happen in practice anyway.